Samples.xcodeproj/project.pbxproj

```diff
@@ -532,22 +534,20 @@
 			name = Products;
 			sourceTree = "<group>";
 		};
-		218F35B229C28F4A00502022 /* Authenticate with OAuth */ = {
+		1CB749EC29CCE8EB00B222AD /* Change viewpoint */ = {
 			isa = PBXGroup;
 			children = (
-				218F35B329C28F4A00502022 /* AuthenticateWithOAuthView.swift */,
+				1CB749EE29CCE8EB00B222AD /* ChangeViewpointView.swift */,
 			);
-			path = "Authenticate with OAuth";
+			path = "Change viewpoint";
 			sourceTree = "<group>";
 		};
-		4D2ADC5329C4F612003B367F /* Change map view background */ = {
+		218F35B229C28F4A00502022 /* Authenticate with OAuth */ = {
 			isa = PBXGroup;
 			children = (
-				4D2ADC5529C4F612003B367F /* ChangeMapViewBackgroundView.swift */,
-				4D2ADC5829C4F612003B367F /* ChangeMapViewBackgroundView.SettingsView.swift */,
-				4D2ADC6129C5071C003B367F /* ChangeMapViewBackgroundView.Model.swift */,
+				218F35B329C28F4A00502022 /* AuthenticateWithOAuthView.swift */,
 			);
-			path = "Change map view background";
+			path = "Authenticate with OAuth";
```

Comment:
Looks like the authentication sample is overwriting the change map view sample. Is that right? The Change map view sample works fine but the app crashes when I tap the info button for reademe/source code.

Reply:
The OAuth problem might be introduced when Nimesh resolved the merge conflict manually, causing the sample order in xcodeproj file to mess up. It's fixed in https://github.com/Esri/arcgis-maps-sdk-swift-samples/pull/129#discussion_r1146620930 , so when this PR merges/rebases the latest `v.next` it should be gone.

The info button crash is fixed in #141